src/github.rs

```diff
@@ -468,6 +471,19 @@ pub enum PullRequestReviewState {
     Pending,
 }
 
+#[derive(Debug, serde::Deserialize, Eq, PartialEq)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum AuthorAssociation {
+    Collaborator,
+    Contributor,
+    FirstTimer,
+    FirstTimeContributor,
+    Mannequin,
+    Member,
+    None,
+    Owner,
```

Comment:
The accepted values can be found on the "Response schema" of the issues endpoint, https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#get-an-issue.

Reply:
Why not use https://docs.rs/octocrab/latest/octocrab/models/enum.AuthorAssociation.html directly?

Reply:
None of the struct or enums in this file uses it, it would be a first. Not opposed but that's just not what we historically did.

Reply:
I would prefer defaulting to using third-party/octocrab code where possible, and only falling back to custom code when we have a specific use-case for it. We could probably delete a lot of code from triagebot if we just started using octocrab more :)

Reply:
Switched to octocrab.